src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -137,6 +142,46 @@ public function getStyle()
         return $this->style;
     }
 
+    /**
+     * Sets table style.
+     *
+     * @param int $columnIndex Column index
+     * @param TableStyle|string $name The style name or a TableStyle instance
+     *
+     * @return Table
+     */
+    public function setColumnStyle($columnIndex, $name)
+    {
+        if (!is_numeric($columnIndex)) {
+            throw new \InvalidArgumentException(sprintf('Invalid column index: "%s".', $columnIndex));
+        }
+
+        if ($name instanceof TableStyle) {
+            $this->columnStyles[$columnIndex] = $name;
+        } elseif (isset(self::$styles[$name])) {
+            $this->columnStyles[$columnIndex] = self::$styles[$name];
+        } else {
+            throw new \InvalidArgumentException(sprintf('Style "%s" is not defined.', $name));
+        }
+
+        return $this;
+    }
+
+    /**
+     * Gets the current style for specified column.
+     *
+     * @var int $columnIndex Column index
+     * @return TableStyle
+     */
+    public function getColumnStyle($columnIndex)
+    {
+        if (isset($this->columnStyles[$columnIndex])) {
+            return $this->columnStyles[$columnIndex];
+        } else {
```

Comment:
Nope, all columns should have current table style by default, if not assigned from outside. If I remove else, rendering will be broken.